src/ResourceManager/AnalysisServices/AzureRM.AnalysisServices.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '0.4.2'
+ModuleVersion = '1.0.0'
```

Comment:
@taiwu please revert this change; we will update the module version as a part of the release